packages/runtime/src/widgets/Row.tsx

```diff
@@ -34,7 +34,7 @@ export const Row: React.FC<FlexboxProps> = (props) => {
     if (!action) {
       return;
     }
-    action.callback();
+    action.callback({ ...parentScope });
```

Comment:
Same thing, scope should be injected by the action. The parent scope should get removed here, it's automatically provided in the custom scope provider.